components/switch/switch-visibility.js

```diff
@@ -32,6 +64,29 @@ class VisibilitySwitch extends LocalizeCoreElement(SwitchMixin(LitElement)) {
 		return html`<d2l-icon icon="tier1:visibility-show"></d2l-icon>`;
 	}
 
+	get _labelContent() {
+		if (this._text) return super._labelContent;
+
+		const tooltipHelpClasses = {
+			'switch-visibility-conditions-show': this.on && this._hasConditions
+		};
+
+		const conditions = html`
+			<d2l-tooltip-help 
+				class="${classMap(tooltipHelpClasses)} d2l-switch-text" 
+				id="conditions-help" 
```

Comment:
nit: (only if you're changing other things) attributes could be in alphabetical order